scripts/circle-ci/run-circle-tests.sh

```diff
@@ -55,7 +57,7 @@ JAVA_GC_LOGGING_OPTIONS="${JAVA_GC_LOGGING_OPTIONS} -XX:-TraceClassUnloading"
 JAVA_GC_LOGGING_OPTIONS="${JAVA_GC_LOGGING_OPTIONS} -Xloggc:build-%t-%p.gc.log"
 
 # External builds have a 16gb limit.
-if [ "$CIRCLE_NODE_INDEX" -eq "7" ]; then
+if [ "$CIRCLE_NODE_INDEX" -eq "9" ]; then
```

Comment:
Why did we change this from 7 to 9?

Reply:
discussed offline: this is the errorprone container, and when we added container 8 we forgot to update this.